br/pkg/checksum/executor.go

```diff
@@ -82,6 +90,7 @@ func (builder *ExecutorBuilder) Build() (*Executor, error) {
 		builder.concurrency,
 		builder.oldKeyspace,
 		builder.newKeyspace,
+		builder.resourceGroupName,
```

Comment:
what's the server side behaviour if the client forget to set resourceGroupName, or the client is malicious?

Reply:
server code is in processing, I think we should provide a fallback mechanism to the default resource group. cc @glorv

Reply:
Currently no extra safety guarantee because tidb-lightning directly visits tikv, this is the same behavior as before. BTW, resource-control only affect the schedule priority, so it does not need more safety check than before.